src/ml_board/backend/dummy_server/dummy_server.py

```diff
@@ -0,0 +1,162 @@
+####################################################################################################
+# imports
+import json
+import sys
+from collections import defaultdict
+from typing import List
+
+from flask import Flask, copy_current_request_context, render_template, request
+from flask_socketio import SocketIO, disconnect, emit, join_room, rooms
+
+####################################################################################################
+# TODO: ASK Max what's that for?
+# class EventSubscriberIF:
+#     def callback(self):
+#         raise NotImplementedError
+
+
+####################################################################################################
+# Flask HTTP Server
+def CreateFlaskServer() -> Flask:
+    # create Flask server
+    _app: Flask = Flask(__name__, template_folder="template")
+    _app.secret_key = 'secret!'
+
+    # set routes with their respective callbacks
+    _app.add_url_rule('/', view_func=_index)
+    # _app.add_url_rule('/status', view_func=_status)
+    # _app.add_url_rule('/api/status', view_func=_api_status)
+
+    return _app
+
+
+def _index() -> str:
+    # return render_template('index.html', async_mode=ws._socketio.async_mode)
+    return "Index Page"
+
+
+# def _status() -> str:
+#     return render_template('status.html', async_mode=ws._socketio.async_mode)
+
+
+# def _api_status() -> dict:
+#     client_sids = ws.client_sids
+#     room_key_to_sid = defaultdict(list)
+#     for client_sid in client_sids:
+#         room_keys = rooms(client_sid, "/")
+#         for room_key in room_keys:
+#             room_key_to_sid[room_key].append(client_sid)
+#     return {"clients": client_sids, "rooms": room_key_to_sid}
+
+# TODO: RESTful APIs could be added here!
```

Comment:
this should be another PR and be documented in an issue rather than in code. But the idea is good :+1: